__tests__/TimelineResultCompTest.js

```diff
@@ -56,7 +56,7 @@ describe("TimelineResultComp", () => {
 
 	it("contains all div components of the timeline table", function() {	
```

Comment:
instead of finding just all divs, can you find an item more specific for user timeline and then the same for filter timeline?